scripts/btsbot/README.md

```diff
@@ -0,0 +1,4 @@
+# TODO:
+- [ ] Add image scale (1.01" per pixel for ZTF)
+- [ ] Obtain PSFs and incorporate into dataset
+- [ ] Implement same train/val/test splits as BTSbot dataset
```

Comment:
Thanks for looking over things!
Short answer: no, but I think it might be okay to leave these as to-dos for v0 (and can complete them in a separate PR in future).
2. So eventually I would like to get the PSF files for all the images (it will be a fair bit of work and isn't essential information for the intended use of this dataset), but I checked with Francois and apparently it should be okay to omit this info for the version 0 of AstroPile.
1. I found out that the image scale of ZTF images is 1.01" per pixel. I could include this easily now, but was going to do it at the same time as (2).
3. For now I was going to agglomerate all of train/val/test into train, and the user can define their own splits for now.

Reply:
Omit psf file is fine for now.
Include as float. 
Split as before if possible. 👍

Reply:
Include wavelength / band feature.

Reply:
Adding band and scale now resolved.
Adding train/val/test splits is resolved on the data loader end, but not the hdf5 file saving end.

Reply:
All complete